src/utils/fullScreenWidthPixels.js

```diff
@@ -1,6 +1,12 @@
 import devicePixelRatio from "./devicePixelRatio";
 
+const SSR_FALLBACK_WIDTH_PX = 945;
```

Comment:
Could you add your commit comment in code? Not every reader will easily find the commit that introduced this number.